### PR TITLE
Some suggestions for sentences in README.md

### PR DESCRIPTION
**Pull request by arrrggghhh**
Hi,

I made some changes in README.

I think it would help users more if you specify what are the additional features and how to use them in the document.

Thank you for your work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,24 @@
 # SwiftySensorControl
 
-**SwiftySensorControl** helps to use various sensors' data instead of [CMMotionManager](https://developer.apple.com/library/ios/documentation/CoreMotion/Reference/CMMotionManager_Class/).
+**SwiftySenserControl** is an alternative for [CMMotionManager](https://developer.apple.com/library/ios/documentation/CoreMotion/Reference/CMMotionManager_Class/).
 
-1. [Why SwiftySensorControl?](## Why is the SwiftySensorControl useful?
-)
-1. [What kind of sensors are there?](#What-kind-of-sensor-datas-are-there-in-SwiftySensorControl?)
+1. [Why SwiftySensorControl?](## Why SwiftySensorControl?)
```
Comment (JeffGuKang) [inline README.md]: Link would be not active.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,24 @@
...
+1. [What kind of data can SwiftySensorControl read?](## What kind of data can SwiftySensorControl read?)
```
Comment (JeffGuKang) [inline README.md]: Link would be not active also.

**Comment by JeffGuKang** [comment]
Thanks for your help. 
If some wrong links are update, I could apply your support.

**Comment by arrrggghhh** [comment]
I fixed in-page links.

Thanks :)

**Comment by JeffGuKang** [comment]
I merged. Thank you :)

**Pull request merged**